api/response/schema_query.py

```diff
@@ -178,5 +186,15 @@ def resolve_all_room_responses(
             lower_limit = None
             responses = Query.search_response_by_room(
                 self, info, upper_limit, lower_limit, room)
-
-        return AllResponses(responses=responses)
+        if page and per_page:
+            paginated_response = ListPaginate(
+                iterable=responses, per_page=per_page, page=page)
+            current_page = paginated_response.current_page
+            has_previous = paginated_response.has_previous
+            has_next = paginated_response.has_next
+            pages = paginated_response.pages
+            return PaginatedResponses(responses=current_page,
+                                      has_previous=has_previous,
```

Comment:
@bevkololi please can you shed more light on why responses is equal to the current_page?

Reply:
<img width="554" alt="screenshot 2019-02-19 at 12 40 38" src="https://user-images.githubusercontent.com/26184534/53005135-a5c33e00-3443-11e9-86ab-0c505658c099.png">
@kampkelly, I re-used this function in the `helpers/pagination.py`. it returns the contents of that particular page. Formating the name to suit the responses functionality will affect other paginations in the codebase.

Reply:
Okay, I understand now.